dtfs-central-api/src/services/repositories/utilisation-reports-repo.test.js

```diff
@@ -0,0 +1,42 @@
+const { saveUtilisationReportDetails } = require('./utilisation-reports-repo');
+const db = require('../../drivers/db-client');
+const { DB_COLLECTIONS } = require('../../constants/db_collections');
```

Comment:
Looks like this needs to end `require('../../constants/dbCollections');` now.

Reply:
VSCode not smart enough :(